icicle/include/api/babybear.h

```diff
@@ -56,6 +56,9 @@ extern "C" cudaError_t babybear_mul_cuda(
 extern "C" cudaError_t babybear_add_cuda(
   babybear::scalar_t* vec_a, babybear::scalar_t* vec_b, int n, vec_ops::VecOpsConfig& config, babybear::scalar_t* result);
 
+extern "C" cudaError_t babybear_accumulate_cuda(
+  babybear::scalar_t* vec_a, babybear::scalar_t* vec_b, int n, vec_ops::VecOpsConfig& config);
+
```

Comment:
I don't see the extension definition for babybear and the vec_ops_ext header is not excluded in the generation script. Any idea why it wasn't generated?

Reply:
hi - as we discussed on daily few days ago and as @ChickenLover  suggested - it would be great to regenerate headers from script, however I tried and seems like it's worth separate pr as maybe won't be relevant to v3? and the templates differ significantly including formatting seems like quite few methods were added manually. I tried to fix them but it's a bit imho overkill for this pr and worth separate if will be actual, wdyt?